Makefile

```diff
@@ -44,6 +44,12 @@ ifeq ($(CEPH_VERSION),ci-pacific)
 	GO_CEPH_VERSION := pacific
 	BUILD_TAGS := pacific,ceph_dev_pacific
 endif
+ifeq ($(CEPH_VERSION),ci-main)
```

Comment:
> Just a personal preference but since Ceph's main branch will always be pre-release, I think we can drop the "ci-" prefix here. It'll save me a tiny bit of typing since we don't need to distinguish release "quincy" versus pre-release "ci-quincy".

Yup, make sense.

> I'd also be open to other prefixes or suffixes here if people have opinions on it ("ci/quincy" "quincy+ci"? Whatever is fine tho :-) )

How about `pre-{quincy, pacific, octopus}`?

Reply:
> > Just a personal preference but since Ceph's main branch will always be pre-release, I think we can drop the "ci-" prefix here. It'll save me a tiny bit of typing since we don't need to distinguish release "quincy" versus pre-release "ci-quincy".
> 
> Yup, make sense.
> 
> > I'd also be open to other prefixes or suffixes here if people have opinions on it ("ci/quincy" "quincy+ci"? Whatever is fine tho :-) )
> 
> How about `pre-{quincy|pacific|octopus}`? We could then use similar build tags(`ceph_pre_quincy`, `ceph_pre_pacific`, `ceph_pre_octopus`) in sources too.

Sounds good to me!